en/docs/references/org-domains-urls.md

```diff
@@ -1,24 +1,80 @@
-# Organization domains and URLs
+# Organization-Specific REST API Routing
 
 ## Organization URLs
 
-As a part of the B2B Organization Management feature, incorporated the organization identifier as first-class support for the URLs (similar to the current tenant specific endpoints) by introducing a path param to denote the specific organization to which the request should be directed to.
+The Management Console URL in the WSO2 CIAM Cloud contains a path parameter to identify the organization. This denotes the specific organization to which requests should be routed.
 
-URL format - o/{organization id}/…
+URL format - `o/{organization id}/…`
 
-Organization-specific routing capability is introduced. The BasicAuthenticationHandler has also been improved to handle authentication when the username has the organization domain(id) appended.
+Organization-specific routing in WSO2 CIAM Cloud supports both basic authentication and OAuth 2.0 flows as explained below.
 
-The current behavior in BasicAuthenticationHandler is that if the username doesn't have the organization domain, the user is authenticated against the ROOT organization.
+## Basic Authentication
+
+The `BasicAuthenticationHandler` has been improved to handle authentication when the username has the organization domain(id) appended.
+
+For example, consider the following request to get the organization list:
+
+``` java
+curl GET 'https://localhost:9443/o/<org-id>/api/server/v1/organizations' \
+--header 'accept: application/json' \
+--header 'Authorization: Basic [Base64encode(Username>:<Password>)]'
+```
+
+If the username in the `BasicAuthenticationHandler` doesn't have the organization domain, the user is authenticated against the super organization.
 It was decided to improve the authentication logic as given below.
--   If the username doesn't have the organization domain, the user is authenticated against the /o/<org>.
--   Else retrieve the organization domain from the username (break from the last @ in the username) and authenticate the user against that organization.
+
+<img src="../../../assets/img/references/organization-specific-rest-api-routing/improved_basicauthenticationhandler.png" alt="Improved BasicaAthenticationHandler" width="700">
+
+-   If the username doesn't have the organization domain, the user is authenticated against the `/o/<org>`.
```

Comment:
```suggestion
If the username doesn't have the organization domain, the user is authenticated against `/o/<org>`. Otherwise, you can retrieve the organization domain from the username (note the break with the last `@` in the username) and authenticate the user against that organization.
```